lib/matobj2.gd

```diff
@@ -1433,17 +1433,17 @@ DeclareOperation( "IsBound[]", [ IsRowListMatrix, IsPosInt ] );
 
 #############################################################################
 ##
-#O  Unbind\[\]( <mat>, <pos> )
+#O  Unbind\[\]( <M>, <pos> )
 ##
 ##  <#GAPDoc Label="RowListMatObj_Unbind">
 ##  <ManSection>
-##  <Oper Name="Unbind\[\]" Arg='mat, pos' Label="for a row list matrix"/>
+##  <Oper Name="Unbind\[\]" Arg='M, pos' Label="for a row list matrix"/>
 ##
 ##  <Returns>nothing</Returns>
-##
+##a
```

Comment:
This extra 'a' is confusing GAPDoc, once this is cleaned up I think everything will work!